org.eclipse.winery.lsp/src/test/resources/tosca-community-contributions/mwe/service_template_v3.yaml

```diff
@@ -0,0 +1,9 @@
+tosca_definitions_version: 2.0
```

Comment:
Remove the files org.eclipse.winery.lsp/src/test/resources/tosca-community-contributions/mwe/* for now.

Are they used for automated tests or for manual tests?

If for automated tests, we need to first have an IP review with Eclipse team.